opentracing-util/src/test/java/io/opentracing/util/AutoFinishScopeManagerTest.java

```diff
@@ -57,4 +63,36 @@ public void activateSpan() throws Exception {
         Scope missingSpan = source.active();
         assertNull(missingSpan);
     }
+
+    @Test
+    public void captureScope() throws Exception {
+        Span span = mock(Span.class);
+
+        // We can't use 1.7 features like try-with-resources in this repo without meddling with pom details for tests.
```

Comment:
I suggest we consider dropping support for 1.6.

Reply:
+1, about time.

Reply:
Agree, but hopefully in a separate PR?

Reply:
I think a discussion is appropriate but I'm not sure we should hold 0.32 for it.

Reply:
I think we can discuss that as part of #218 and decide what to do from there - and I agree with @austinlparker, I think we shouldn't hold 0.32 for this change to happen ;)